src/apis/setAnnotationContentOverlayHandler.js

```diff
@@ -0,0 +1,22 @@
+import action from 'actions';
+
+/**
+ * Adds a custom overlay to annotations if that annotation currently support it. Currently only the Ellipsis annotation supports it.
+ * @method WebViewerInstance#setAnnotationContentOverlayHandler
+ * @param {function} customOverlayHandler a function that takes an annotation and returns a DOM Element, which is rendered as a tooltip when hovering over the annotation
+ *  * @example
+WebViewer(...)
+  .then(function(instance) {
+    instance.setAnnotationContentOverlayHandler(annotation => {
+        var div = document.createElement("div")
```

Comment:
Let's make the code snippets a bit more consistent. Can we use `const`, `let`, single quote for strings, and end the function call with a `;`?